APCAppCore/APCAppCore/Library/MedicationTrackingSetup/MedicationFrequency/APCFrequencyEverydayTableViewCell.m

```diff
@@ -35,10 +35,6 @@
 
 @implementation APCFrequencyEverydayTableViewCell
 
-- (void)awakeFromNib
-{
-}
-
 - (void)setSelected:(BOOL)selected animated:(BOOL)animated
```

Comment:
is there a reason you removed this pass-through method from the other cells' .m files but not this one?